src/pages/Notifications/List/BehaviorGroupsSection.tsx

```diff
@@ -276,21 +214,23 @@ export const BehaviorGroupsSection: React.FunctionComponent<BehaviorGroupSection
               </>
             )}
           </Stack>
-          {editModalState.isOpen && (
-            <BehaviorGroupWizardPage
-              bundle={props.bundle}
-              applications={props.applications}
-              behaviorGroup={editModalState.template}
-              onClose={onCloseModal}
-            />
-          )}
-          {deleteModalState.isOpen && (
-            <DeleteBehaviorGroupPage
-              behaviorGroup={deleteModalState.data}
-              onClose={onCloseDelete}
-            />
-          )}
-        </ExpandableSection>
+        </div>
+
+        {editModalState.isOpen && (
+          <BehaviorGroupWizardPage
+            bundle={props.bundle}
+            applications={props.applications}
+            behaviorGroup={editModalState.template}
+            onClose={onCloseModal}
+          />
+        )}
+        {deleteModalState.isOpen && (
+          <DeleteBehaviorGroupPage
+            behaviorGroup={deleteModalState.data}
+            onClose={onCloseDelete}
+          />
+        )}
+        {/* </ExpandableSection> */}
```

Comment:
Delete the commented code. Looks good otherwise

Reply:
oops! done!